L/LibUV_jll/Compat.toml

```
@@ -2,7 +2,8 @@
 JLLWrappers = "1.1.0-1"
 
 ["2.0.0"]
```

```suggestion
["2.0.0-2.0.0"]
```

@staticfloat Is this what you meant?